undertow/src/main/resources/org/wildfly/extension/undertow/LocalDescriptions.properties

```diff
@@ -169,6 +169,7 @@ undertow.listener.allow-equals-in-cookie-value=If this is true then Undertow wil
 undertow.listener.no-request-timeout=The length of time in milliseconds that the connection can be idle before it is closed by the container.
 undertow.listener.request-parse-timeout=The maximum amount of time (in milliseconds) that can be spent parsing the request
 undertow.listener.rfc6265-cookie-validation=If cookies should be validated to ensure they comply with RFC6265.
+undertow.listener.allow-unescaped-characters-in-url=If Undertow should accept non-encoded characters that are disallowed by the URI specification.
```

Comment:
Maybe `Whether` instead of `If` at the beginning would be better. Also - we might include some kind of risk warning as this is something users rather should avoid?